src/rpc/handlers/AccountMPTokenIssuances.cpp

```diff
@@ -73,6 +73,24 @@ AccountMPTokenIssuancesHandler::addMPTokenIssuance(
     setFlag(issuance.mptCanTransfer, ripple::lsfMPTCanTransfer);
     setFlag(issuance.mptCanClawback, ripple::lsfMPTCanClawback);
 
+    if (sle.isFieldPresent(ripple::sfMutableFlags)) {
+        auto const mutableFlags = sle.getFieldU32(ripple::sfMutableFlags);
+
+        auto const setMutableFlag = [&](std::optional<bool>& field, std::uint32_t mask) {
+            if ((mutableFlags & mask) != 0u)
+                field = true;
+        };
+
+        setMutableFlag(issuance.mptCanMutateCanLock, ripple::lsmfMPTCanMutateCanLock);
+        setMutableFlag(issuance.mptCanMutateRequireAuth, ripple::lsmfMPTCanMutateRequireAuth);
+        setMutableFlag(issuance.mptCanMutateCanEscrow, ripple::lsmfMPTCanMutateCanEscrow);
+        setMutableFlag(issuance.mptCanMutateCanTrade, ripple::lsmfMPTCanMutateCanTrade);
+        setMutableFlag(issuance.mptCanMutateCanTransfer, ripple::lsmfMPTCanMutateCanTransfer);
+        setMutableFlag(issuance.mptCanMutateCanClawback, ripple::lsmfMPTCanMutateCanClawback);
+        setMutableFlag(issuance.mptCanMutateMetadata, ripple::lsmfMPTCanMutateMetadata);
+        setMutableFlag(issuance.mptCanMutateTransferFee, ripple::lsmfMPTCanMutateTransferFee);
```

Comment:
Maybe it would be good to add a parametrised test to check output for each single flag presented in `mutableFlags`. What do you think?